lapce-ui/src/palette.rs

```diff
@@ -319,7 +324,9 @@ impl Widget<LapceTabData> for PaletteContainer {
         if old_data.palette.input != data.palette.input
             || old_data.palette.index != data.palette.index
```

Comment:
I think if you do

```
old_data.palette.input != data.palette.input
            || old_data.palette.index != data.palette.index
            || old_data.palette.run_id != data.palette.run_id
```

Then you don't need the ensure_item_visible in lifecycle.

I might be wrong. Can you please test.

Reply:
Oh yeah that's much cleaner, thanks!